[2024-03-09T15:50:28.373+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop manual__2024-03-09T10:20:01+00:00 [queued]>
[2024-03-09T15:50:28.378+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop manual__2024-03-09T10:20:01+00:00 [queued]>
[2024-03-09T15:50:28.378+0530] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-09T15:50:28.407+0530] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): run_after_loop> on 2024-03-09 10:20:01+00:00
[2024-03-09T15:50:28.410+0530] {standard_task_runner.py:60} INFO - Started process 5060 to run task
[2024-03-09T15:50:28.415+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2024-03-09T10:20:01+00:00', '--job-id', '13', '--raw', '--subdir', '/home/kai/.local/lib/python3.10/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp4ye8rr8v']
[2024-03-09T15:50:28.416+0530] {standard_task_runner.py:88} INFO - Job 13: Subtask run_after_loop
[2024-03-09T15:50:28.503+0530] {task_command.py:423} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2024-03-09T10:20:01+00:00 [running]> on host BreakThrough.
[2024-03-09T15:50:28.596+0530] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='run_after_loop' AIRFLOW_CTX_EXECUTION_DATE='2024-03-09T10:20:01+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-09T10:20:01+00:00'
[2024-03-09T15:50:28.597+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-09T15:50:28.598+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 1']
[2024-03-09T15:50:28.612+0530] {subprocess.py:86} INFO - Output:
[2024-03-09T15:50:28.613+0530] {subprocess.py:93} INFO - 1
[2024-03-09T15:50:28.614+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-09T15:50:28.646+0530] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20240309T102001, start_date=20240309T102028, end_date=20240309T102028
[2024-03-09T15:50:28.708+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-09T15:50:28.720+0530] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
